generators/parser.go

```diff
@@ -0,0 +1,58 @@
+package generators
```

Comment:
This should have a package comment. 

If you haven't already, set up lint tools for your editor, or run [golint](https://github.com/golang/lint) / [golangci-lint](https://golangci-lint.run/) manually.